### PR TITLE
add publicKey in `Address` type

### PR DESCRIPTION
**Pull request by louisinger**
add the member `publicKey: string` in `Address` type: bip32 generated public key associated with the address derivation path (hex encoded).

it closes #15

### CODE REVIEW

**Review by Shuhaib07 — COMMENTED** [review]
approved

File: src/types.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export type Address = {
   unconfidentialAddress?: string;
   contract?: Contract;
   script: string;
+  publicKey: string; // the public key associated with the derivation path
```
Comment (aoaei) [inline src/types.ts:51]: 1NNAP4ZYwn63KHZXG5vPaP26W3ixdFSw45

Comment (aoaei) [inline src/types.ts:51]: 6DA6BA6388BAC005

**Review by aoaei — COMMENTED** [review]
1NNAP4ZYwn63KHZXG5vPaP26W3ixdFSw45

**Comment by aoaei** [comment]
> สมาชิก`publicKey`

6DA6BA6388BAC005

**Pull request merged**